### PR TITLE
Test NFS rootsquash cli option

### PR DESCRIPTION
**Pull request by msainiRedhat**
# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Comment by msainiRedhat** [comment]
Logs - http://magna002.ceph.redhat.com/cephci-jenkins/msaini/nfs/rootsquash/cepci-log/

**Comment by pranavprakash20** [comment]
LGTM

File: tests/nfs/test_export_rootsquash.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        # Check permission of file created by squashed user
+        clients[0].exec_command(
+            sudo=True, cmd=f"ls -n {nfs_squash_mount}/file_squashed"
+        )
+        out = get_file_owner(f"{nfs_squash_mount}/file_squashed", clients)
+        if "squashuser" not in out:
+            log.error("File is not created by squashed user")
+            return 1
+        log.info("File created by squashed user")
+
```
Comment (MohitBis) [inline tests/nfs/test_export_rootsquash.py:123]: No except block

Comment (msainiRedhat) [reply tests/nfs/test_export_rootsquash.py:123]: Added

**Review by pranavprakash20 — APPROVED** [review]
LGTM

**Review by MohitBis — APPROVED** [review]
LGTM

**Review by vdas-redhat — APPROVED** [review]
LGTM

**Pull request merged**